verification/src/transaction_verifier.rs

```diff
@@ -946,3 +1020,21 @@ where
         Ok(fee)
     }
 }
+
+/// log tx verification result into monitor log
+struct MonTxVerifyStatus {
```

Comment:
It's a bit of overskill. It's OK to just add a simple function to print the log:

```rust
fn log_tx_verification_result(hash: &H256, cycles: Cycle) {
    trace_target!("ckb_tx_monitor", r#"{{"tx_hash":"{}","cycles":{}}}"#, hash, cycles);
}
```

This code snippet also uses raw string literals to make the code clean.

Reply:
yes, of course.